docs/prod-host-setup.md

```diff
@@ -328,3 +328,51 @@ wget -O - https://meltdown.ovh | bash
 
 [1]: https://elixir.free-electrons.com/linux/v4.14.203/source/virt/kvm/arm/hyp/timer-sr.c#L63
 [2]: https://lists.cs.columbia.edu/pipermail/kvmarm/2017-January/023323.html
+
+### Linux 6.1 boot time regressions
+
+Linux 6.1 introduced some regressions[^1] in the time it takes to boot a VM, for
+the x86_64 architecture. They can be mitigated depending on the CPU and the
+version of cgroups in use.
+
+[^1]: See
+explanation in commit 8d2745383b0e4e0196bab8492f796fbf2e402a98
```

Comment:
This commit hash looks not on the firecracker upstream.
https://github.com/firecracker-microvm/firecracker/commit/8d2745383b0e4e0196bab8492f796fbf2e402a98

Reply:
Yeah I just realized every time I update it, it will change since it tries to refer to itself. Not sure what can be done here beyond amend it afterwards?

Reply:
or we could say "on this commit's message"?

Reply:
Maybe we can invert the dependency here? E.g. put the full description here and have the commit message refer to the md file?

Reply:
Yep, that is the most expedient route